ezyrb/mapper.py

```diff
@@ -0,0 +1,243 @@
+"""
+Utilities for mapping solutions on a different grid
```

Comment:
Yeah, you're right, I apologize. My english is horrible, and also I wrote the documentation so much (too much) quickly. I'll recheck everything, at least in mapper class, as soon as possible.